### PR TITLE
BatchNorm is done

### PR DESCRIPTION
**Pull request by smolix**
*Description of changes:*

Replaced with Lazy versions of tensor. Conversion to overall training code is still outstanding.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1962/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1962/
